### PR TITLE
Fix #79

### LINKED ISSUE #79: Muted state for chats
As mentionned on the API doc https://docs.pushbullet.com/#update-chat
I'm creating a Gtk+ application for Pushbullet and i would love to implement this as a feature :)
Thanks!

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**

### CODE REVIEW

**Comment by kovacsbalu** [comment]
Hi! @bil-elmoussaoui can you please cleanup your code. You probaly want to fix #79 but is has other modifications also.

**Comment by bil-elmoussaoui** [comment]
I cant do that now. My laptop's sceeen is broken! I will try to do that as
soon as possible. What else i have to change?

File: pushbullet/pushbullet.py [hunk 1]
```diff
@@ -161,8 +161,8 @@ def edit_chat(self, chat, name):
             raise PushbulletError(r.text)
 
 
-    def mute_chat(self, chat):
-        data = {"muted":  True}
+   def mute_chat(self, chat):
```
Comment (kovacsbalu) [inline pushbullet/pushbullet.py]: fix indent 4 space

**Comment by kovacsbalu** [comment]
6920a2f and 0fdc83c commit shows some modification in encryption, but I can see the diff now :)
https://github.com/randomchars/pushbullet.py/compare/master...bil-elmoussaoui:patch-1

**Comment by bil-elmoussaoui** [comment]
I will fix that right now :+1:

**Comment by bil-elmoussaoui** [comment]
I will close this and create a new PR right now

**Pull request closed without merging**